api/spec/factories/groups.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
```

Comment:
Fijate lo que te comenté sobre `shoulda-matchers` 🙂 
Si al final cambiás el test de la `time_preference`... entonces la Factory no sería necesaria por ahora ([Rails YAGNI](https://railshurts.com/2015/principles/#yagni---you-arent-gonna-need-it))